gateway/src/cluster/impl.rs

```diff
@@ -249,6 +249,17 @@ impl Cluster {
             .cloned()
     }
 
+    /// Returns a list of all the shards
```

Comment:
`Return` at the beginning and a period at the end to be consistent with the other methods please